docker_launch.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+if [ ! -f ./security.yml ]; then
+    ./security-setup --enable=false
+fi
+
+cat > wait-for-hosts.yml << EOF
```

Comment:
maybe just place a playbook named `wait-for-hosts.yml` within the playbooks directory, and then the path would be `/mi/playbooks/wait-for-hosts.yml`